Overview.md

```diff
@@ -0,0 +1,18 @@
+# **Microsoft Azure and the Call Center Industry – A Vital Partnership for Business Growth**
+## An application of the callcenterapp program
+--- 
+Many businesses around the world employ customer call centers as an integral part of their operation. Call centers are facilities that receive, process information, and make calls to existing or potential customers. These call centers are the vital link between the company and its  current and future customers. For example, a tech company might employ a call center to help with any issues that customers  of their products  might have. Call center employees might also reach out to potential future customers through ‘cold calls’. 
+In spite of this important role, the call center industry today still remains highly manual, employing many people to service requests. This is disadvantageous to companies that employ call centers because they need to spend a significant part of their budget paying, training and retaining call center employees. Furthermore, the cost of maintaining and operating a call center itself poses a significant financial strain. The cumulative capital spent on call centers could be better used elsewhere, like in improving the quality of products or investing in new business propositions. Automating the call center industry would also eliminate the human cost of abuse that call center agents sometimes face on a daily basis.. Serverless computer farms provide a possible solution to the problem by allowing companies to use cloud-based programs to deal with customers. Together with other automated programs like Cortina for front end human interface, such farms can also provide analytic capability and collection of data that would otherwise have to be documented by hand. Microsoft Azure is the leading serverless computer farm on the market, and it allows its clients to run software on the cloud, and process data more easily to run their business. Businesses can deploy resources like CPUs, storage, and applications in a more scalable manner with Azure without any upfront significant investment, allowing them to scale-up as they grow. Furthermore, Azure has a function to provide advice to companies about how to increase sales, something that cannot be easily done without expert knowledge. If marketed the right way and customized for the call center industry, Microsoft Azure will have an opportunity to make a profound impact on how call centers are run and operated. . 
```

Comment:
```suggestion
Despite this important role, the call center industry today remains highly manual, employing many people to service requests. This is disadvantageous to companies that use call centers because they need to spend a significant part of their budget paying, training, and retaining call center employees. Furthermore, the cost of maintaining and operating a call center itself poses a significant financial strain. Automating the call center industry would also eliminate the human cost of abuse that call center agents sometimes face daily. Serverless computer farms provide a possible solution to the problem by allowing companies to use cloud-based programs to deal with customers. Together with other automated programs like Cortana for a front end human interface, such farms can also provide analytic capability and collection of data that would otherwise have to be documented by hand. Microsoft Azure allows its clients to run software on the cloud, and process data more easily to run their business. Businesses can deploy resources like CPUs, storage, and applications in a more scalable manner with Azure without any significant upfront investment, allowing them to scale-up as they grow. Furthermore, Azure has a function to provide advice to companies about how to increase sales, something that cannot be easily done without expert knowledge. If cloud applications are produced for the call center industry, Microsoft Azure will have an opportunity to make a profound impact on how call centers are run and operated. 
```